vertx-sql-client/src/main/java/io/vertx/sqlclient/SqlResultProperty.java

```diff
@@ -0,0 +1,7 @@
+package io.vertx.sqlclient;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+@VertxGen
+public interface SqlResultProperty<T> {
```

Comment:
I think we should name it more generically so it could eventually be reused for another similar case, I think naming it `PropertyKind` is good.